pkg/runtime/wfengine/wfengine.go

```diff
@@ -210,6 +214,10 @@ func (wfe *WorkflowEngine) Stop(ctx context.Context) error {
 	wfe.startMutex.Lock()
 	defer wfe.startMutex.Unlock()
 
+	if !wfe.IsRunning {
```

Comment:
It looks to me like we removed all assignments to `wfe.IsRunning`, but we still check it here and in `Start(...)`. Did I miss something?

Reply:
Whoops that was a mistake. WHile working on this I made some changes and then didn’t revert them correctly. I’ll fix this tomorrow. Thanks for catching it

Reply:
Fixed